test/TestInfrastructure/TestExtensions/TestDefaultConfiguration.cs

```diff
@@ -32,6 +32,12 @@ public static void InitializeDefaults()
         public static string EventHubConnectionString => defaultConfiguration[nameof(EventHubConnectionString)];
         public static string ZooKeeperConnectionString => defaultConfiguration[nameof(ZooKeeperConnectionString)];
 
+        public static bool GetValue(string key, out string value)
+        {
+            value = defaultConfiguration[key];
+            return value != null;
+        }
```

Comment:
This will throw if the value isn't found, but the return value suggests it will return false instead